solutions.txt

```diff
@@ -67,8 +62,7 @@ Exercise 6:
 
 foods.splice(1, 1, sushi, cupcake)
```

Comment:
hm, sushi and cupcake should be strings, so that the strings can be inserted into the array.

Reply:
this is similar to cheeseburger earlier